build.sbt

```diff
@@ -319,15 +319,14 @@ lazy val docs = http4sProject("docs")
     },
     copySiteToStage := {
       streams.value.log.debug(s"copying ${preStageSiteDirectory.value} to ${siteStageDirectory.value}")
-
       IO.copyDirectory(
         source = preStageSiteDirectory.value,
         target = siteStageDirectory.value,
         overwrite = false,
         preserveLastModified = true)
       IO.copyDirectory(
         source = tutTargetDirectory.value,
-        target = siteStageDirectory.value / "content" / "v0.16-cats",
+        target = siteStageDirectory.value / "content" / "v${apiVersion.value._1}.${apiVersion.value._2}",
```

Comment:
Not sure if it's here or somewhere else, but when I `rm -rf docs/target` and then `sbt makeSite`, I end up with:
```
$ ls docs/target/hugo | grep apiver
vapiversion.value._1.apiversion.value._2/
```
Seems to go away with rossabaker/http4s#2 .